docs/content/integrations/dbt/reference.mdx

```diff
@@ -31,6 +31,9 @@ For a step-by-step implementation walkthrough, refer to the [Using dbt with Dags
 | <PyObject module="dagster_dbt" object="get_asset_key_for_model" />                             | A helper method that retrieves the <PyObject module="dagster" object="AssetKey" /> for a dbt model.                                                 |
 | <PyObject module="dagster_dbt" object="get_asset_key_for_source" />                            | A helper method that retrieves the <PyObject module="dagster" object="AssetKey" /> for a dbt source with a singular table.                          |
 | <PyObject module="dagster_dbt" object="get_asset_keys_by_output_name_for_source" />            | A helper method that retrieves the <PyObject module="dagster" object="AssetKey" />'s for a dbt source with multiple tables.                         |
+| <PyObject module="dagster_dbt" object="load_assets_from_dbt_project" />                        | Deprecated in favor of <PyObject module="dagster_dbt" object="dbt_assets" decorator />.                                                             |
```

Comment:
I think we could leave this out

Reply:
+1 to updating the API docs and then leaving this out

Reply:
Could you clarify what change we should make in the API docs for `load_assets_from_dbt_project`? We already recommend `@dbt_assets` there. https://github.com/dagster-io/dagster/blob/28391da99fc62b7996e2a84994ae73029d5e59d2/python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py#L503-L509